test/test_multiprocessing.py

```diff
@@ -939,4 +940,6 @@ def test_is_shared_cuda(self):
 
 
 if __name__ == '__main__':
-    run_tests()
+    # multiprocessing ipc use case not supported on Jetson
```

Comment:
Would it make sense to add a warning here if `IS_JETSON` is true? Otherwise this would silently skip all tests which diverges from typical behavior